DataBinding/DataBindingAssemblyInfo.cs

```diff
@@ -0,0 +1,33 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2016 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+using System.Reflection;
+
+[assembly: AssemblyDescription("Realm is a mobile database: a replacement for SQLite")]
+[assembly: AssemblyCopyright("Copyright © 2017 Realm")]
```

Comment:
I read recently that the year you put in a copyright statement is supposed to represent the earliest year from which the copyright applies. Thus, it should not be updated. I am not sure if that info was valid and I don't think it really matters as *at least* half of the web then gets this wrong, but since we're being overly pedantic on the .net team, I thought I'd bring it up.. :-)